server/__init__.py

```diff
@@ -26,3 +34,48 @@ async def convert_settings_overrides(
         return await super().convert_settings_overrides(
             source_version, overrides
         )
+
+    async def get_simple_actions(
+        self,
+        project_name: str | None = None,
+        variant: str = "production",
+    ) -> list[SimpleActionManifest]:
+        """Return a list of simple actions provided by the addon"""
+        output = []
+
+        # Add 'Create Project Folder Structure' action to folders.
+        output.append(
+            SimpleActionManifest(
+                identifier=f"{IDENTIFIER_PREFIX}.create_project_structure",
+                label="Create Project Folder Structure",
+                icon={
+                    "type": "material-symbols",
+                    "name": "create_new_folder",
+                },
+                order=100,
+                entity_type="folder",
+                entity_subtypes=None,
+                allow_multiselection=False,
+            )
+        )
+
+        return output
+
+    async def execute_action(
+        self,
+        executor: ActionExecutor,
+    ) -> ExecuteResponseModel:
+        """Execute webactions."""
+
+        project_name = executor.context.project_name
+
+        if executor.identifier == \
+              f"{IDENTIFIER_PREFIX}.create_project_structure":
+            return await executor.get_launcher_action_response(
+                args=[
+                    "create-project-structure",
+                    "--project", project_name,
+                ]
+            )
```

Comment:
I believe there'll always be a `project_name` here. 
But, in case there is not, what should I return 
```suggestion
        if executor.identifier == \
              f"{IDENTIFIER_PREFIX}.create_project_structure":
            
            if not project_name:
                # Do something
                return

            return await executor.get_launcher_action_response(
                args=[
                    "create-project-structure",
                    "--project", project_name,
                ]
            )
```

Reply:
I did it anyways https://github.com/ynput/ayon-core/pull/1193/commits/0ec82bd34d83542949ab27ae209c012082714916